### PR TITLE
Update showcase.json

### PR DESCRIPTION
**Pull request by jakepeg**
Added DooCoins

Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @jakepeg,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/380) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -2900,5 +2900,28 @@
...
+},
+  {
+  "name": "DooCoins",
+  "description": "Reward your kids with DooCoins for their good behaviour, completing chores, and personal achievements.\n",
+  "website": "https://www.doo.co",
+  "logo": "/img/showcase/doocoins_logo.webp",
+  "screenshots": [
+    "/img/showcase/doocoins_screenshot.webp"
+  ],
+  "video": "",
```
Comment (jennifertrin) [inline showcase.json:2912]: Can you please remove `Video` in the JSON if you do not have a video?

File: showcase.json [hunk 2]
```diff
@@ -2900,5 +2900,28 @@
...
+  "display": "Normal",
+  "id": "doocoins",
+  "oneLiner": "Kids rewards dApp",
+  "stats": "PWA",
```
Comment (jennifertrin) [inline showcase.json:2916]: Can you please advise what `PWA` means? Can you please provide a number stat on usage if available?

**Review by jennifertrin — CHANGES_REQUESTED** [review]

**Comment by jennifertrin** [comment]
> Dear @jakepeg,
> 
> In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).
> 
> If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/380) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.
> 
> — The DFINITY Foundation

Can you please also go through this CLA step?

File: showcase.json (hunk 2 above)
Comment (jakepeg) [reply showcase.json:2916]: I've removed stats

File: showcase.json (hunk 1 above)
Comment (jakepeg) [reply showcase.json:2912]: Video removed

**Comment by jennifertrin** [comment]
@jakepeg I will close this PR as we already addressed these changes as part of #2014 and #2016.

**Comment by jakepeg** [comment]
Great, thank youOn 20 Oct 2023, at 06:11, Jennifer K. Tran ***@***.***> wrote:﻿
Closed #2013.

**Pull request closed without merging**